.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,47 @@
+name: enclave-cc e2e tests
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc install e2e tests
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build and push to local registry
+        uses: docker/build-push-action@v3
```

Comment:
This is something I wasn't aware of, and maybe we should explore of using more of it in different places. :-)